src/constants.ts

```diff
@@ -65,3 +65,9 @@ export enum ScmType {
     LocalGit = 'LocalGit',
     GitHub = 'GitHub'
 }
+
+export enum PythonAlias {
```

Comment:
nit: This is only used in the PythonProjectCreator file, right? Seems like that's a better place to put this